schemas/2015-04-01/Microsoft.NotificationHubs.json

```diff
@@ -58,6 +58,47 @@
       ],
       "description": "Microsoft.NotificationHubs/namespaces resource"
     },
+    "namespaces_AuthorizationRules": {
```

Comment:
>namespaces_AuthorizationRules [](start = 5, length = 29)

I guess you need to update 2015-01-01/deploymentTemplate.json to include the new resource types.